symfony/maker-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "bundles": {
+        "Symfony\\Bundle\\MakerBundle\\MakerBundle": ["dev"]
+    },
+    "aliases": ["maker", "generator"]
```

Comment:
What about also adding `code-generator` (as used in the keywords of the bundle) and maybe also `generator-bundle`?

Reply:
Aliases should be short and sweet.

Reply:
generator-bundle would be confusing as the current one generates CRUD, not this one. So, make-bundle is still not a replacement for SensioGeneratorBundle (unfortunately).